### PR TITLE
YR-Fix-documentation/FeedRecordeduture/XSUP-29069

### PR DESCRIPTION
**Pull request by RosenbergYehuda**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
relates: https://jira-dc.paloaltonetworks.com/browse/XSUP-29069
## Description
- Added documentation to the user to avoid receiving 100k indicators per instance, since we are limited by the API to have only 100k indicators active.

- change the default expiration method to be by indicator type, instead of "when removed from feed" to gain a little more time before those extra indicators expire.
Note:
The "extend Docker timeout" in the name of the branch was not implemented in the end, so please disregard it.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/FeedRecordedFuture/ReleaseNotes/1_0_31.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### Recorded Future RiskList Feed
+- Updated the integration description to include a notice advising users against fetching 100,000 or more indicators per instance.
```
Comment (JasBeilin) [inline Packs/FeedRecordedFuture/ReleaseNotes/1_0_31.md:5]: You also changed the default value for expiration, lets add that.

**Review by JasBeilin — CHANGES_REQUESTED** [review]
Looks great, please fix the one tiny issue and we can move forward.

File: Packs/FeedRecordedFuture/ReleaseNotes/1_0_31.md (hunk 1 above)
Comment (RosenbergYehuda) [reply Packs/FeedRecordedFuture/ReleaseNotes/1_0_31.md:5]: Nice catch!

**Review by JasBeilin — APPROVED** [review]
Great!

File: Packs/FeedRecordedFuture/Integrations/FeedRecordedFuture/README.md [hunk 2]
```diff
@@ -38,8 +39,8 @@ This integration was integrated and tested with Recorded Future Feed
     | Hash | 1 Day. |
     | URL | 2 Hours. |
     | Vulnerability | 2 Hours. |
-3. Per instance configuration, it is recommended to use either `connectApi` or `fusion` as a service for chosen indicator type, and not both, as most of the data between both services is duplicated.
-4. The feed size can be change according to the chosen indicator type:
+4. Per instance configuration, it is recommended to use either `connectApi` or `fusion` as a service for chosen indicator type, and not both, as most of the data between both services is duplicated.
+5. The feed size can be change according to the chosen indicator type:
```
Comment (ShirleyDenkberg) [inline Packs/FeedRecordedFuture/Integrations/FeedRecordedFuture/README.md:43]:
```suggestion
5. The feed size can be changed according to the chosen indicator type:
```

File: Packs/FeedRecordedFuture/Integrations/FeedRecordedFuture/README.md [hunk 3]
```diff
@@ -92,3 +93,8 @@ To limit the 'connectApi' service indicators list.
 | RecordedFutureFeed.RiskRule.Name | String | The risk rule name. | 
 | RecordedFutureFeed.RiskRule.Description | String | The risk rule description. | 
 | RecordedFutureFeed.RiskRule.Criticality | String | The risk rule criticality. | 
+
+
+### Troubleshooting
+
+If indicators expire unexpectedly, please ensure that the feed is not receiving more than 100,000 indicators per fetch. As it is discouraged to use "large" as a risk rule, we currently receive indicators in a single large CSV file containing up to 100,000 indicators. If Recorded Future has additional indicators to send, the CSV will be sorted in descending order based on the highest score. Consequently, some indicators may not pass through, leading to their expiration, particularly if the expiration is configured as "When removed from the feed" and they were present in our system from previous fetches.
```
Comment (ShirleyDenkberg) [inline Packs/FeedRecordedFuture/Integrations/FeedRecordedFuture/README.md:100]:
```suggestion
If indicators expire unexpectedly, ensure that the feed is not receiving more than 100,000 indicators per fetch. As it is discouraged to use "large" as a risk rule, we currently receive indicators in a single large CSV file containing up to 100,000 indicators. If Recorded Future has additional indicators to send, the CSV will be sorted in descending order based on the highest score. Consequently, some indicators may not pass through, leading to their expiration, particularly if the expiration is configured as "When removed from the feed" and they were present in our system from previous fetches.
```

**Comment by ShirleyDenkberg** [comment]
@JasBeilin Doc review completed.

**Pull request merged**